.circleci/config.yml

```diff
@@ -490,8 +490,9 @@ jobs:
                         name: Bump git version
                         working_directory: tools
                     - run:
-                        command: tools/create-jira-release create-release ${NEW_VERSION_NAME} --private-key ${JIRA_PRIVATE_KEY} --access-token ${JIRA_OAUTH_TOKEN} --consumer-key ${JIRA_CONSUMER_KEY} --project-name ${CIRCLE_PROJECT_REPONAME}
+                        command: yarn jira-release create ${NEW_VERSION_NAME} --private-key ${JIRA_PRIVATE_KEY} --access-token ${JIRA_OAUTH_TOKEN} --consumer-key ${JIRA_CONSUMER_KEY} --project-name ${CIRCLE_PROJECT_REPONAME}
```

Comment:
create-jira-release would be a better naming.

Reply:
Imo not otherwise you have `create-jira-release create` which is duplicated.

Reply:
you are right. overlooked it.